### ISSUE TITLE
Adding new one-electron integral in libcint and use it in pyscf

### ISSUE THREAD

**Issue by jiachenchem** [pyscf/pyscf#374]
I am trying to add a new one-electron integral in libcint, and use it in pyscf. What I did was to edit 
/home/jia/test_sofwater/pyscf/pyscf/lib/build/deps/src/libcint/scripts/auto_intor.cl by adding one line 
'("int1e_test"                ( \| nabla-rinv \| ))
Then, clisp auto_intor.cl; mv *.c ../src/autocode/
cd /home/jia/test_sofwater/pyscf/pyscf/lib/build; make
Error message is like below:
/home/jia/test_sofwater/pyscf/pyscf/lib/build/deps/src/libcint/src/autocode/intor1.c:3173:9: error: ‘g2’ undeclared (first use in this function); did you mean ‘g1’?
 G1E_D_I(g2, g0, envs->i_l+0, envs->j_l+0, 0);
...

I would like to know what is the good practice to add one-electron integral. Appreciate your help.

**Comment by hebrewsnabla** [comment]
but the latest libcint release say:
"Add integral < i | nabla-rinv | j >"

**Comment by jiachenchem** [comment]
Thank you for letting me know, I tried the new version 3.0.18 and < i | nabla-rinv | j > is working. I would like thank Sunqm for the work.

**Comment by sunqm** [comment]
It's good that you noticed the changes in the integral code. The updates in the integral library will be available in the next release of pyscf (1.7). The function name will be mol.intor('int1e_drinv'). In versions older than 1.7, below is a workaround:
```
a = mol.intor('int1e_iprinv', comp=3)
result = a + a.transpose(0,2,1)
```

**Comment by sunqm** [comment]
Available in pyscf-1.7
